src/Features/Core/Portable/CodeFixes/CodeFixService.cs

```diff
@@ -430,6 +430,17 @@ private ImmutableArray<DiagnosticId> GetFixableDiagnosticIds(CodeFixProvider fix
             }
         }
 
+        private ImmutableArray<string> GetAndTestFixableDiagnosticIds(CodeFixProvider f)
+        {
+            var ids = f.FixableDiagnosticIds;
+            if (ids.IsDefault)
+            {
+                throw new ArgumentException(WorkspacesResources.FixableDiagnosticIdsIncorrectlyInitialized, nameof(f.FixableDiagnosticIds));
```

Comment:
It's weird for the ArgumentException to have the ParamName of "FixableDiagnosticIds", which isn't a parameter name for this method or the caller above (GetFixableDiagnosticIds), right?